### DISCUSSION TITLE
Status bar branch list shows deleted branches

### DISCUSSION THREAD

**Discussion by lukaszbienkowski-ttc** [atom/atom#23541] [Q&A]
Hi,
I have been using Atom for a few months now for a project, and created 50+ branches since. I try to keep it tidy in github and delete the branches that no longer serve me. However, atom doesn't seem to pick up these deletions and keeps on showing me all the branches that ever existed. Is there any way to clean the list?

**Comment by icecream17** [comment]
git doesn't update locally unless you tell it to

so run some prune command to prune deleted branches

https://www.git-tower.com/learn/git/faq/cleanup-remote-branches-with-git-prune

**Comment by lukaszbienkowski-ttc** [comment]
That's great. Thanks. I also needed to remove local branches that are no longer in use, which I did using this command
`git branch --v | grep "\[gone\]" | awk '{print $1}' | xargs git branch -D`
following advice in here https://stackoverflow.com/a/59228595
